src/NewTwitchApi/Cli/IO/Stdin.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace NewTwitchApi\Cli\IO;
+
+class Stdin
+{
+    public function read(): string
+    {
+        return fgets(STDIN);
```

Comment:
It's nit-picky but I wonder if it might be nicer to trim here and not here: https://github.com/ChessCom/twitch-api-php/pull/1/files#diff-0aafca4c8dde7eb43ce69af0e4696be8R18

Reply:
I considered this, but in the end, this class is simply an abstraction around reading input, so that's all it does. It's up to the code that uses this class to decide whether or not to trim. Looking at it another way, if you were to come into this code base fresh and start coding and used `Stdin::read()`, would you assume it is automatically trimming for you? I would not, so I've left this simply an IO abstraction.